tests/daemon_test_fixture.cpp

```diff
@@ -471,8 +471,8 @@ std::string mpt::DaemonTestFixture::fake_json_contents(const std::string& defaul
         mount_element += gid_array_elements.join(',');
 
         mount_element += QString::fromStdString(fmt::format("\n                ],\n"
-                                                            "                \"source_path\": \"{}\",\n"
-                                                            "                \"target_path\": \"{}\",\n",
+                                                            "                \"source_path\": {:?},\n"
+                                                            "                \"target_path\": {:?},\n",
```

Comment:
1. Without the debug specifier the JSON on Windows will contain `"C:\some\path\"`, since the `\` is not escaped when we parse the JSON the `"` gets escaped and things break. 
2. Yep, their [syntax documentation](https://fmt.dev/11.0/syntax/#format-specification-mini-language) describes the purpose of `'?'`, which is "Debug format. The string is quoted and special characters escaped." So we get `"C:\\some\\path\\"` instead which gets parsed correctly.

Reply:
I see, so the difference between {:?} and {} on a std::string is just the quotes and escapes. {:?} always gives the raw and quoted string, which is definitely more robust.